src/app/zkapp_test_transaction/lib/commands.ml

```diff
@@ -122,9 +122,9 @@ let generate_zkapp_txn (keypair : Signature_lib.Keypair.t) (ledger : Ledger.t)
   let spec =
     { Mina_transaction_logic.For_tests.Transaction_spec.sender =
         (keypair, Account.Nonce.zero)
-    ; fee = Currency.Fee.of_int 10000000000 (*1 Mina*)
+    ; fee = Currency.Fee.of_mina_int_exn 10
```

Comment:
Should it be 1 mina here ? It seems there is a mismatch between the comments and the code at the moment. It should not be blocking for this PR though.

Reply:
The comment said `1 Mina`, but the actual value spelled `10` mina instead. I retained the value used in these tests previously and deleted the misleading comment.